extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -498,7 +498,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	public async searchJupyterBooks(treeItem?: BookTreeItem): Promise<void> {
 		let folderToSearch: string;
 		if (treeItem && treeItem.sections !== undefined) {
-			folderToSearch = treeItem.uri ? getContentPath(treeItem.version, treeItem.book.root, path.dirname(treeItem.uri)) : getContentPath(treeItem.version, treeItem.book.root, '');
+			folderToSearch = treeItem.uri ? getContentPath(treeItem.book.version, treeItem.book.root, path.dirname(treeItem.uri)) : getContentPath(treeItem.book.version, treeItem.book.root, '');
```

Comment:
So it seems like we have an issue here then - since BookTreeItem sets its version to book.version when it's created if it's a book. 

https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/src/book/bookTreeItem.ts#L55

So is the issue then that this is broke for any non-book items? (since it looks like we never set the version there)

Does it even make sense for each BookTreeItem to have its own version vs just always using the book.version?